deploy/vagrant/vbox/buildbox.sh

```diff
@@ -12,9 +14,16 @@ if [ -f tachyon-dev.box ]; then
   rm -f tachyon-dev.box
 fi
 
-vagrant up
-vagrant package --output tachyon-dev.box default
-vagrant destroy -f
+BASE_BOX='chef/centos-7.0'
+AVAIL_BOX=$(vagrant box list | grep $BASE_BOX | grep '(virtualbox, 1.0.0)')
+
```

Comment:
My understanding is that vagrant will only run provisioner once. If relaunched, it will take no action -- this is different than docker.

If this is the case, the old code wasted time at the first launch by invoking vagrant up twice. First it provisions python, java, maven in the vm then save it. Then it runs vagrant again, and provision these things via ansible one more time. I have already removed the first provisioning (see TACHYON-804). And now I am trying to create tachyon-dev box using cli. 

We actually can use chef/centos-7.0 directly, but I think resave it to tachyon-dev box will allow us to package a well provisioned box later. By "well", I mean everything, tachyon and ufs. Those new boxes should be releasted directly later on, which will be surely launched faster.